tests/rptest/tests/tx_verifier_test.py

```diff
@@ -8,15 +8,20 @@
 # by the Apache License, Version 2.0
 
 from rptest.services.cluster import cluster
-from ducktape.mark import ignore
 from ducktape.errors import DucktapeError
 
 from rptest.tests.redpanda_test import RedpandaTest
-from rptest.clients.kafka_cli_tools import KafkaCliTools
 from rptest.clients.rpk import RpkTool
 
 import subprocess
 
+# Expected log errors in tests that test misbehaving
+# transactional clients.
+TX_ERROR_LOGS = [
+    # e.g. cluster - rm_stm.cc:370 - Can't prepare pid:{producer_identity: id=1, epoch=27} - unknown session
+    "cluster - rm_stm.*unknown session"
```

Comment:
@rystsov please could you review this part -- currently the all_tx_tests triggers this error which I presume is the result of a client doing something like timing out a session and then trying to use it?

I wonder if the message could be improved a bit to make it more obvious to a user what the error is about (at present it doesn't mention transactions)